src-tauri/src/main.rs

```diff
@@ -71,7 +63,7 @@ async fn farming(path: String, reward_address: String, plot_size: u64) {
 }
 
 #[tauri::command]
-async fn start_node(path: String) -> FarmerIdentity {
+async fn start_node(path: String) -> [u8; 32] {
```

Comment:
Does it still need to return anything with recent changes?

Reply:
unfortunately yes, the conversion for the publicAddress format conversion on the frontend side requires localApi.
It sounds bad, I know. But no worries, after this PR merges, this whole address thing needs to be revized in the multi-farming setting (each plot having a public address). That's my next todo, so I vote for leaving this as is atm.